matcher/helpers.go

```diff
@@ -173,10 +173,14 @@ func machineEventuallyConnects(m types.Machine, t ...int) {
 	}, time.Duration(time.Duration(dur)*time.Second), time.Duration(5*time.Second)).Should(Equal("ping\n"))
 }
 
-func machineReboot(m types.Machine) {
+func machineReboot(m types.Machine, t ...int) {
 	machineSudo(m, "reboot") //nolint:errcheck
 	time.Sleep(1 * time.Minute)
-	machineEventuallyConnects(m, 750)
+	timeout := 750
+	if len(t) != 0 {
+		timeout = t[0]
```

Comment:
I'm probably missing something but I don't see why t needs to be more than a single int if only the first one will be used anyway